lib/addons/jupyterhub/index.ts

```diff
@@ -135,7 +135,6 @@ export class JupyterHubAddOn extends HelmAddOn {
         this.options = this.props as JupyterHubAddOnProps;
     }
     
-    @dependable(AwsLoadBalancerControllerAddOn.name)
```

Comment:
GitHub action warning: unused import of dependable.

Reply:
addressed.